hibernate-core/src/main/java/org/hibernate/dialect/CUBRIDDialect.java

```diff
@@ -87,6 +88,11 @@ public CUBRIDDialect() {
 		registerKeyword( "SEARCH" );
 	}
 
+	@Override
+	public int getVersion() {
+		return 0;
+	}
+
```

Comment:
I do think a version field is deserved to be included into Dialect base class. I even created a PR for this previously: https://github.com/hibernate/hibernate-orm/pull/3316.